gluoncv/utils/export_helper.py

```diff
@@ -98,11 +99,23 @@ def export_block(path, block, data_shape=None, epoch=0, preprocess=True, layout=
     # try different data_shape if possible, until one fits the network
     last_exception = None
     for dshape in data_shapes:
-        h, w, c = dshape
+        if layout in ('HWC', 'CHW'):
+            h, w, c = dshape
```

Comment:
I think HWC and CHW need to be separated, otherwise it's confusing.
Previously I didn't take them into consideration which is bad though.

Reply:
Yes, I agree. Will separate them.